Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackParamsEstimationAlgorithm.hpp

```diff
@@ -69,20 +69,13 @@ class TrackParamsEstimationAlgorithm final : public IAlgorithm {
     std::shared_ptr<const Acts::MagneticFieldProvider> magneticField;
     /// The minimum magnetic field to trigger the track parameters estimation
     double bFieldMin = 0.1 * Acts::UnitConstants::T;
-    /// Constant term of the loc0 resolution.
-    double sigmaLoc0 = 25 * Acts::UnitConstants::um;
-    /// Constant term of the loc1 resolution.
-    double sigmaLoc1 = 100 * Acts::UnitConstants::um;
-    /// Phi angular resolution.
-    double sigmaPhi = 0.02 * Acts::UnitConstants::degree;
-    /// Theta angular resolution.
-    double sigmaTheta = 0.02 * Acts::UnitConstants::degree;
-    /// q/p resolution.
-    double sigmaQOverP = 0.1 / Acts::UnitConstants::GeV;
-    /// Time resolution.
-    double sigmaT0 = 10 * Acts::UnitConstants::ns;
+    /// Initial covariance matrix diagonal.
+    std::array<double, 6> initialSigmas = {
+        1 * Acts::UnitConstants::mm,     1 * Acts::UnitConstants::mm,
+        1 * Acts::UnitConstants::degree, 1 * Acts::UnitConstants::degree,
+        1 / Acts::UnitConstants::GeV,    1 * Acts::UnitConstants::ns};
```

Comment:
Should we keep the previous defaults?

Reply:
right lets keep the defaults and physmon. I will adopt the values in https://github.com/acts-project/acts/pull/2295